Client/Frontend/Fakespot/Views/FakespotOptInCardView.swift

```diff
@@ -0,0 +1,359 @@
+
+            contentStackView.topAnchor.constraint(equalTo: mainView.topAnchor,
+                                                  constant: UX.contentStackViewPadding),
+            contentStackView.bottomAnchor.constraint(equalTo: mainView.bottomAnchor,
+                                                     constant: -UX.contentStackViewPadding),
+            contentStackView.leadingAnchor.constraint(equalTo: mainView.leadingAnchor,
+                                                      constant: UX.contentStackViewPadding),
+            contentStackView.trailingAnchor.constraint(equalTo: mainView.trailingAnchor,
+                                                       constant: -UX.contentStackViewPadding),
+        ])
+    }
+
+    // MARK: Button Actions
+    @objc
+    private func didTapLearnMore() {
+        viewModel?.onTapLearnMore()
+    }
+
+    @objc
+    private func didTapTermsOfUse() {
+        viewModel?.onTapTermsOfUse()
+    }
+
+    @objc
+    private func didTapPrivacyPolicy() {
+        viewModel?.onTapPrivacyPolicy()
+    }
+
+    @objc
+    private func didTapMainButton() {
+        viewModel?.shouldOptIn = true
+        viewModel?.onTapMainButton()
+    }
+
+    @objc
+    private func didTapSecondaryButton() {
+        
+        viewModel?.onTapSecondaryButton()
+    }
+
+    // MARK: View Setup
+    func configure(_ viewModel: FakespotOptInCardViewModel) {
+        self.viewModel = viewModel
+
+        headerLabel.text = viewModel.headerTitleLabel
+        headerLabel.accessibilityIdentifier = viewModel.headerLabelA11yId
+
+        bodyFirstParagraphLabel.attributedText = getFirstParagraphText()
+        bodyFirstParagraphLabel.accessibilityIdentifier = viewModel.bodyFirstParagraphA11yId
+
+        disclaimerTextLabel.attributedText = getDisclaimerText()
+        disclaimerTextLabel.accessibilityIdentifier = viewModel.disclaimerTextLabelA11yId
+
+        learnMoreButton.setTitle(viewModel.learnMoreButtonTitle, for: .normal)
+        learnMoreButton.accessibilityIdentifier = viewModel.learnMoreButtonTitleA11yId
+
+        termsOfUseButton.setTitle(viewModel.termsOfUseButtonTitle, for: .normal)
+        termsOfUseButton.accessibilityIdentifier = viewModel.termsOfUseButtonTitleA11yId
+
+        privacyPolicyButton.setTitle(viewModel.privacyPolicyButtonTitle, for: .normal)
+        privacyPolicyButton.accessibilityIdentifier = viewModel.privacyPolicyButtonTitleA11yId
+
+        mainButton.setTitle(viewModel.mainButtonTitle, for: .normal)
+        mainButton.accessibilityIdentifier = viewModel.mainButtonTitleA11yId
+
+        secondaryButton.setTitle(viewModel.secondaryButtonTitle, for: .normal)
+        secondaryButton.accessibilityIdentifier = viewModel.secondaryButtonTitleA11yId
+
+        let cardModel = ShadowCardViewModel(view: mainView, a11yId: viewModel.cardA11yId)
+        cardContainer.configure(cardModel)
+    }
+
+    func getFirstParagraphText() -> NSAttributedString {
```

Comment:
can be private? 
```suggestion
    private func getFirstParagraphText() -> NSAttributedString {
```

same comment for `getFirstParagraphWebsites` and `getDisclaimerText`